src/Mailer/Handler/SendMailsForOwnershipEventHandler.php

```diff
@@ -37,6 +37,10 @@ public function handle(DomainMessage $domainMessage): void
             return;
         }
 
+        if ($domainMessage->getMetadata()->get('disable_mails')) {
```

Comment:
I see we already have some code on line 36 to check for a replay, maybe we need to follow the same pattern and implement a new `DomainMessageSpecificationInterface`?

Reply:
Done.